### ISSUE TITLE
Kodi NFO metadata for images are not local paths

### ISSUE THREAD

**Issue by quadcom** [Radarr/Radarr#8399]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Radarr downloads the posters and fanart to the media location (movie folder). But it names the files as "poster.jpg" and "fanart.jpg." This is not in line with the naming conventions that Kodi uses to identify objects.

https://kodi.wiki/view/NFO_files/Movies#:~:text=the%20above%20traps.-,The,-below%20image%20details

Kodi is looking for images with the "<movie_name>-<image_purpose>.jpg" format in the Single Movie file type. Blueray and DVD folders have different requirements, which do align with how Radarr currently stores the extras content. But I believe that +90% of the content people have is of the Single Movie type.

This may have been overlooked and not noticed, as the NFO file that Radarr creates doesn't include the local image path and name to be scraped but rather an online URL for the image to be downloaded.."

### Expected Behavior

I had expected that when the images are downloaded and the system is set to store Kodi-compatible metadata, it would follow the Kodi standard for naming those files. 

I also expected that the NFO file would refer to those locally stored images to define the poster and fanart available for Kodi to use.

### Steps To Reproduce

Set the system metadata output for Kodi/XMBC

Import an existing movie into the library.

It will import and save the poster and fanart images with the incorrect naming convention. And the NFO will contain URLs to download the images, forcing Kodi to download them again.

I tested it on my system three times with an existing movie (Love The Coopers). I deleted it from Radarr and cleared out all the extras that were added to the media folder. On import, Radarr parsed the file and downloaded the extras as before, naming the images incorrectly. I also confirmed the NFO file again has URLs to the image locations online rather than the local path.

### Environment

```markdown
- OS: Unraid/Docker
- Radarr: 4.5.0.7106-ls94 by linuxserver.io
- Docker Install: Yes
- Using Reverse Proxy: No
- Browser: Chrome
- Database: Internal
```

### What branch are you running?

Master

### Trace Logs?

[Radarr trace - Love The Coopers.txt](https://github.com/Radarr/Radarr/files/11344978/Radarr.trace.-.Love.The.Coopers.txt)

![2023-04-27 10_51_59-Love the Coopers (2015) 333348-tt2279339](https://user-images.githubusercontent.com/7085931/234901519-28d3603d-4463-4783-b807-3d137b8664fb.jpg)

**Comment by bakerboy448** [comment]
Naming is a duplicate of https://github.com/Radarr/Radarr/issues/5050

**Comment by thezoggy** [comment]
`poster.jpg` - only works if you have stored your movies in separated folder AND have the Movies are stored in separated folders matching the moviename checked

`<moviename>-poster.jpg` - works if you store movies in separated folders or dont (like a monster).


Your wanting radarr to grab the images and understand how they should be stored and denote it to align in the movie nfo file (if enabled)?

**Comment by quadcom** [comment]
Hey Zoggy,

_The long story that goes on a bit of a tangent._

I've been using Kodi for years now, and only just starting to get under the hood of it. Sonarr and Radarr are really new to me too but I am loving the ecosystem they provide. Kudos to the whole team on this project! 

But the RRR's and Kodi don't seem to want to work together as seamlessly as they could. It appears that the Servarr ecosystem has chosen Plex as its primary target of front end support. Which is great, I use a Plex server too. But only for mobile streaming, not home use on the TV.

While I understand that Sonarr-extended and Radarr-extended are not supported here. The scripts that grab trailers and extras don't put the DL's in the right folder structure for Kodi to consume and give access to on the front end. I've searched for a solution for some time now and haven't really gotten anywhere. The Kodi "Extras" add-on will look in an "Extras" folder next to the media and all the move/tv extras are to be kept there but the scripts don't do that. They save them in the root of the objects folder. I'm not adept enough at coding to change in the script myself.

**TLDR:**

The Kodi docs were a little confusing/conflicting as the file naming structure listed, was later contradicted with different information. Mainly these poster and fanart image names/locations. Using a single-folder structure, naming the images 'poster.jpg' and 'fanart.jpg' is functional for Kodi to scrape.
